samples/04_gis_analysts_data_scientists/fighting_california_forest_fires_using_spatial_analysis.ipynb

```diff
@@ -30,7 +30,7 @@
   },
```

Comment:
<p>Looks like we need to update the statements here for sharing_level since the code is changed below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>